hubedit/src/main.rs

```diff
@@ -33,6 +33,9 @@ pub enum Command {
         #[clap(short, long)]
         version: String,
 
+        #[clap(short, long)]
+        epoch: Option<String>,
```

Comment:
`Option<u32>`?

Should this be optional at all, or should it be mandatory (like `version`)?

Reply:
I'd like to change this to mandatory after changes to `sprot-release` procedures implement  reviewable release plans.

Reply:
...that will probably be worked into `sprot-release` PR#11 or a follow-on.